themes/frontierline/functions.php

```diff
@@ -662,8 +663,11 @@ function frontierline_disable_emojis() {
  */
 function frontierline_image_reminder() {
   global $pagenow;
-  $message = __('Remember to include a featured image! It should be at least 1400 by 770 pixels. Posts without a featured image will show a standard placeholder image.', 'frontierline');
-  if ($pagenow === 'post-new.php' && (get_theme_mod('frontierline_no_post_thumbnail') !== '1')) {
+  $message = __('Remember to include a featured image! It should be at least 1400 by 770 pixels.', 'frontierline');
+  if (get_theme_mod('frontierline_no_summary_image') !== '1') :
+    $message .= __(' Posts without a featured image will show a standard placeholder image.', 'frontierline');
+  endif;
+  if ($pagenow === 'post-new.php' && get_theme_mod('frontierline_no_post_image') !== '1' ) {
```

Comment:
The message should probably display if either option is enabled (right now it's hidden if summary images are enabled but posts disabled).